service/src/main/java/bio/terra/profile/app/configuration/PolicyServiceConfiguration.java

```diff
@@ -42,16 +43,13 @@ public String getClientCredentialFilePath() {
     return clientCredentialFilePath;
   }
 
-  public void setAzureControlPlaneEnabled(Boolean azureControlPlaneEnabled) {
-    this.azureControlPlaneEnabled = azureControlPlaneEnabled;
-  }
-
-  public Boolean getAzureControlPlaneEnabled() {
-    return azureControlPlaneEnabled;
+  @Autowired
+  PolicyServiceConfiguration(AzureConfiguration azureConfiguration) {
+    this.azureConfiguration = azureConfiguration;
   }
 
   public String getAccessToken() throws IOException {
-    if (azureControlPlaneEnabled) {
+    if (azureConfiguration.controlPlaneEnabled()) {
```

Comment:
Now I see why the controlPlaneEnabled was originally here. This is probably out of scope for this PR, but I think it would be better not to inject one configuration into another. We might need to put this getAccessToken into separate utility so we can inject (or even pass as a param) only AzureConfiguration instead. To be clear what I mean is to 1) move getAccessToken into utility class 2) Inject AzureConfiguration into TpsApiDispatch and pass required flag to generate token. @marctalbott what do you think?